client/src/js/components/bhAddItem.js

```diff
@@ -0,0 +1,35 @@
+angular.module('bhima.components')
+  .component('bhAddItem', {
+    templateUrl : 'modules/templates/bhAddItem.tmpl.html',
+    controller  : addItemController,
+    transclude  : true,
+    bindings    : {
+      itemIncrement    : '<',
+      recipient        : '<',
```

Comment:
I don't see the value of `recipient` ... it looks like just another way to `disable` the input.  Since you already have the `disable` parameter, I propose you remove `recipient`.